server/JsDbg.Gdb/JsDbg.py

```diff
@@ -28,9 +29,10 @@ def __init__(self):
         self.showStderr = False
         self.verbose = True
         # TODO: assume that jsdbg is installed in "~/.jsdbg/" or some other known location?

```

Comment:
This is okay for now, but we should figure out a better solution or add documentation before we ship.

Reply:
Yes, I definitely agree